### DISCUSSION TITLE
Support AWS xray

### DISCUSSION THREAD

**Discussion by kidi** [kamon-io/Kamon#924] [Q&A]
We're using AWS environment and I wonder if some AWS support could be add as a dedicated reporter `zipkin-aws` for instance from https://github.com/openzipkin/zipkin-aws with https://github.com/openzipkin/zipkin-aws/tree/master/reporter/reporter-xray-udp

Does anyone have some experience with AWS environment ?
What could be the easiest way to integrate this kind of need in kamon ?

**Comment by jtjeferreira** [comment]
https://github.com/nezasa/kamon-aws/ this is not what you are asking because is not a reporter to send traces to AWS xray, but it might be a building block so that Kamon can work with xray header format...

**Reply by kidi** [reply]
ah nice, thank for the hint
I didn't think about this option, from what I understand with this lib I could generate traces headers, if I could add support for udp in the zipkin reporter it should do the trick, right?

I am not an expert in trace format between zipkin/jaeger and aws xray.
there multiple version of zipkin from what I began to check with zipkinv2 (do not really know which is used by kamon)
